addons/mail/static/src/models/user_setting.js

```diff
@@ -173,10 +173,10 @@ registerModel({
          * @private
          */
         _loadLocalSettings() {
-            const voiceActivationThresholdString = this.env.services.local_storage.getItem(
+            const voiceActivationThresholdString = this.messaging.browser.localStorage.getItem(
                 "mail_user_setting_voice_threshold"
             );
-            const audioInputDeviceId = this.env.services.local_storage.getItem(
+            const audioInputDeviceId = this.messaging.browser.localStorage.getItem(
```

Comment:
`localStorageService.getItem` does `JSON.parse()`, which made a typecast (e.g. write `10.5` => `"10.5"`, read `"10.5"` => `10.5`).

`windowLocalStorage.getItem` simply returns string. So in example above it returns `"10.5"`.